cli/deployment/api_client_test.go

```diff
@@ -24,3 +24,27 @@ func TestGetOutputs(t *testing.T) {
 		t.Errorf("expected: %s, got: %s", expected, outputs["my-network-prod.name"])
 	}
 }
+
+/*
```

Comment:
@kopachevsky What's the purpose of this commented-out code? It shouldn't be included.

Reply:
@morgante this test uses real API calls, that why I "ignored" it with comments, will remote this